tests/test_oml/test_datasets/test_list_dataest.py

```diff
@@ -0,0 +1,19 @@
+import torch
+
+from oml.const import MOCK_DATASET_PATH
+from oml.datasets.base import ListDataset
+
+
+def test_dataset_len() -> None:
+    images = list((MOCK_DATASET_PATH / "images").iterdir())
+    dataset = ListDataset(images)
+
+    assert len(dataset) == len(images)
+
+
+def test_dataset_iter() -> None:
```

Comment:
please, iterate over dataset with `Dataloader`. And check the shape of the batch is `ndim==4` and that all dims are not zero.